src/Process/PlanarTileProcessing.js

```diff
@@ -9,47 +8,24 @@ export function planarCulling(node, camera) {
     return !frustumCullingOBB(node, camera);
 }
 
-function computeNodeSSE(camera, node) {
-    var vFOV = camera.FOV * Math.PI / 180;
+function _isTileBigOnScreen(camera, node) {
+    const onScreen = camera.box3DSizeOnScreen(node.geometry.OBB.box3D, node.matrixWorld);
 
-    const tmp1 = new THREE.Vector3();
-    tmp1.setFromMatrixPosition(node.matrixWorld);
-    const tmp2 = new THREE.Vector3();
-    tmp2.setFromMatrixPosition(camera.camera3D.matrixWorld);
-    var diff = tmp2.sub(tmp1);
-    const dim = node.extent.dimensions();
-
-    var d = Math.max(0.1, diff.length() - (new THREE.Vector3(dim.x, dim.y, dim.z)).length() * 0.5);
-    var height = 2 * Math.tan(vFOV / 2) * d;
-
-    var dot = diff.normalize().z;
-
-    var ratio = (dim.x * dot) / height;
+    const dim = {
+        x: 0.5 * (onScreen.max.x - onScreen.min.x),
+        y: 0.5 * (onScreen.max.y - onScreen.min.y),
+    };
 
-    if (ratio >= 0.25) {
-        return 7;
-    }
-    return 1;
+    return (dim.x > 0.3 && dim.y >= 0.3);
```

Comment:
Can you elaborate on the magic constants (0.3 here and the multiplication by 0.5 above) and about the `>` for x and `>=` for y?

`0.3` is the equivalent of the old `threshold`, isn't it?